styles/globals.css

```diff
@@ -1,19 +1,63 @@
 @import '../components/Hero/hero.css';
 @import '../node_modules/bootstrap/dist/css/bootstrap.min.css';
```

Comment:
Instead of using css... would you like to use SCSS?

I think it would be good for you to learn.... its the same, but just makes things tidier and easier.
https://nextjs.org/docs/messages/install-sass